README.md

```diff
@@ -79,11 +79,17 @@ Clone this repository to your desired folder: (https://github.com/HERMON-1995/Le
   git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223366027-3a964f33-b6ac-44ea-a6c2-e7fa4683c437.png)

**[Optional]**

- [ ] It would be nice if you remove this Live Demo section for now because you don't have any live link for your application :thinking: When you would have a live link then you can add this Live Demo section with the live link to make your readme file descriptive :+1: